resources/views/home.blade.php

```diff
@@ -1,5 +1,322 @@
+                            <span
+                                class="font-medium">{{__("content.section_4.subtitle_2")}}</span></div>
+                        <ul class="list-disc-none py-8 hidden md:block duration-1000 taos:translate-y-48 taos:opacity-0"
+                            data-taos-offset="100">
+                            @foreach($caseStudy as $key => $description)
+                                <li><span class="font-semibold">{{ $key }}</span> : {{ $description }}</li>
+                            @endforeach
+                        </ul>
+                        <div class="relative pb-8">
+                            <div
+                                class="relative hidden md:flex z-10 duration-1000 taos:translate-y-48 taos:opacity-0"
+                                data-taos-offset="0">
+                                <x-secondary-button href="">{{ __("buttons.all_projects") }}</x-secondary-button>
+                                <x-primary-button href="">{{ __("buttons.case_study") }}</x-primary-button>
+                            </div>
+                            <span class="absolute -left-[30%] -z-10">
+                                <x-bubble size="30"/>
+                            </span>
+                            <span class="absolute -right-[10%] -z-10">
+                                <x-bubble size="80"/>
+                            </span>
+                        </div>
+                    </div>
+                </div>
+
+                <span class="absolute -left-[40%] top-72 -z-10">
+                    <x-bubble size="90"/>
+                </span>
+                <span class="absolute -left-[25%] top-96 -z-10">
+                    <x-bubble size="20"/>
+                </span>
+            </div>
+            <div
+                class="flex rounded-2xl md:bg-gray-light -rotate-3 basis-2/3 relative md:shadow-md size-full left-[5%] lg:left-[10%] mt-12 pb-16 lg:pb-0 delay-200 duration-1000 taos:translate-x-full taos:opacity-0 [animation-iteration-count:infinite]"
+                data-taos-offset="400">
+                <div
+                    class="absolute hidden lg:block rotate-3 z-20 rounded-2xl h-52 w-80 -left-[10%] bottom-[10%] bg-[url('graphics/code-snippet.png')] bg-cover">
+                </div>
+                <img src="graphics/vita.png"
+                     class="w-full relative flex lg:left-[10%] overflow-hidden rounded-2xl bg-white shadow-md rotate-3"
+                     alt="Vita homepage"/>
+
+
+            </div>
+            <div class="flex md:hidden place-content-center px-8">
+                <x-block-primary-button href="">{{ __("buttons.all_projects") }}</x-block-primary-button>
+            </div>
+
+        </section>
+
+        <section class="relative py-20 text-center">
+            <div class="hidden md:block bg-binary min-w-[100%] absolute z-20 inset-0"></div>
```

Comment:
Check where you can use static simply tailwind classes and change in place where you can. In this line you can use  simply `max-w-full` class.  

```suggestion
            <div class="hidden md:block bg-binary min-w-full absolute z-20 inset-0"></div>
```